gridtools/shared/gridtools_utils.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+def file_is_there( check_file: str, debug: bool = False ) -> bool :
+
+    if os.path.isfile( check_file ) :
+        if debug : print(f"File \"{check_file}\" exists")
+        return True
+    else :
+        raise FileNotFoundError(f"Cannot find file \"{check_file}\"")        
```

Comment:
Can we remove the boolean return value from this function? Doesn't seem like it's needed since the error will be raised already.

Reply:
good point, done!